### PR TITLE
Url improvements

### PR DESCRIPTION
**Pull request by fragalcer**
Issues resolved: #

Sanity checks:
- [ ] Have tests been run locally?
- [ ] Does this change any javascript? If so, have you checked that the UI works as expected?
- [ ] If there are data model changes, are relevant data standards referenced - in code?
- [ ] If there are deviations from any specified standards, are the reasons for changes documented?
- [ ] If there are UI changes have you documented whats changed so the help documentation can be updated?

### CODE REVIEW

**Comment by harrywhite4** [comment]
We should look at the use of "path:" since matching the "/" is almost always not what we want

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/slots/tests.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def test_edit_page_permissions(self):
         # Should not be able to view edit page at all
         self.assertRedirects(
             response,
-            reverse('friendly_login') + '?next=/item/' + str(self.newoc.id) + '/edit'
+            reverse('friendly_login') + '?next=' + reverse('aristotle_mdr:edit_item', args=[self.newoc.id])
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/contrib/slots/tests.py]: This is clever!

File: python/aristotle-metadata-registry/aristotle_mdr/urls/aristotle.py [hunk 2]
```diff
@@ -17,173 +17,168 @@
...
+    path('create/', views.create_list, name='create_list'),
+    path('create/wizard/aristotle_mdr/dataelementconcept', views.wizards.DataElementConceptWizard.as_view(), name='createDataElementConcept'),
+    path('create/wizard/aristotle_mdr/dataelement', views.wizards.DataElementWizard.as_view(), name='createDataElement'),
+    path('create/<slug:app_label>/<str:model_name>/', views.wizards.create_item, name='createItem'),
+    path('create/<str:model_name>/', views.wizards.create_item, name='createItem'),
+
+    path('download/options/<slug:download_type>/', views.downloads.DownloadOptionsView.as_view(), name='download_options'),
+    path('download/bulk/<slug:download_type>/', views.downloads.BulkDownloadView.as_view(), name='bulk_download'),
+    path('download/<slug:download_type>/<int:iid>/', views.downloads.DownloadView.as_view(), name='download'),
+    path('dlstatus/<str:taskid>/', views.downloads.DownloadStatusView.as_view(), name='download_status'),
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/urls/aristotle.py]: revert to original regex

later change to `<uuid:taskid>`

**Comment by LegoStormtroopr** [comment]
Looks good. Once changes are made.

File: python/aristotle-metadata-registry/aristotle_mdr/tests/settings/settings.py [hunk 3]
```diff
@@ -76,25 +78,26 @@ def __getitem__(self, item):
...
-    'aristotle_dse',
-    'comet'
-) + INSTALLED_APPS
-
+                     # The good stuff
+                     'aristotle_glossary',
+                     'extension_test',
+                     'aristotle_dse',
+                     'comet',
+                 ) + INSTALLED_APPS
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/tests/settings/settings.py]: Might want to change this back

**Pull request merged**